app/data-sources/ogranisation/applications.js

```diff
@@ -0,0 +1,17 @@
+import { RESTDataSource } from '@apollo/datasource-rest'
+
+export class OrganisationApplicationsDataSourceAPI extends RESTDataSource {
```

Comment:
this should be part of `app/data-sources/rural-payments-portal/RuralPaymentsPortalApi.js`

Reply:
agreed

Reply:
I changed it to rural core API. please see commit: [13dc46](https://github.com/DEFRA/ffc-customer-registry-api/pull/41/commits/13dc46d8f2b1627b4a7bf6cd9b9fce01a86aa658)

Reply:
this done, please see final result, and thank you for pairing to speed up review process